units/systemd-growfs-root.service.in

```diff
@@ -12,7 +12,13 @@ Description=Grow Root File System
 Documentation=man:systemd-growfs-root.service(8)
 
 DefaultDependencies=no
-After=systemd-repart.service systemd-remount-fs.service
+# Partitions must be resized first
+After=systemd-repart.service
+# Make sure the root fs is marked writable before we resize it
+After=systemd-remount-fs.service
+# The file system should be writable before root-fs-ready.target
+Before=root-fs-ready.target
```

Comment:
if root-fs-ready.target should be a passive unit it should be pulled in here, too